src/test.rs

```diff
@@ -116,13 +116,13 @@
 //! stack trace to the line in the test code that panicked: entry 9 in the
 //! stack trace pointing to line 652 of the file `example.rs`.
 
-#[cfg(feature = "use_heap")]
-use crate::bits;
+use crate::{bits, digest, error};
```

Comment:
Should the `bits` module be gated on the `alloc` feature here? Like it is in `lib.rs`?